deploy/templates/service-template.yml

```diff
@@ -80,6 +80,10 @@ parameters:
   - name: MIGRATION_PLANNER_MIGRATIONS_FOLDER
     description: Path to the migration folder containing the sql files used to migrate the db
     value: "/app/migrations"
+  # Envoy Config values
+  - name: ENVOY_IMAGE
+    description: Envoy container image
+    value: docker.io/envoyproxy/envoy:v1.33.2
```

Comment:
I guess we can't use this on app-sre, or do we?

Reply:
Absolutely but I needed creds to pull from there.